### ISSUE TITLE
Try Different Stable Diffusion Fine Tuning For Generation

### ISSUE THREAD

**Issue by flyxiv** [flyxiv/nino#11]
* [x] LoRa
* [x] dreambooth
* [ ] pix2pix
* [ ] controlnet

**Comment by flyxiv** [comment]
# Training Dreambooth
800 epochs for 18 images. Going to take 1.5 hours

![Image](https://github.com/user-attachments/assets/138372ac-763e-479f-a084-ef66ba36443c)

**Comment by flyxiv** [comment]
accelerate launch examples/dreambooth/train_dreambooth.py --pretrained_model_name_or_path="runwayml/stable-diffusion-v1-5" --instance_data_dir=../nino/data/generation/labels --output_dir="C:\dreambooth_model" --instance_prompt="a 2d pixel art sprite of a game character" --class_prompt="a sprite of a game character" --resolution=512 --train_batch_size=1 --gradient_accumulation_steps=1 --learning_rate=1e-5 --lr_scheduler="constant" --lr_warmup_steps=0 --max_train_steps=1000 --checkpointing_steps=500

**Comment by flyxiv** [comment]
instance prompt is too generic. Going to add 'sks' token to uniquely identify the request.

--instance_prompt=a sks pixel art sprite of a game character

**Comment by flyxiv** [comment]
Not generating the way I want. Listing some training prompts of other models

https://huggingface.co/RekklesAI/PokemonStable-v1-5

accelerate launch train_text_to_image.py \
  --pretrained_model_name_or_path="stable-diffusion-v1-5/stable-diffusion-v1-5" \
  --dataset_name="reach-vb/pokemon-blip-captions" \
  --caption_column="text" \
  --image_column="image" \
  --resolution=512 \
  --random_flip \
  --train_batch_size=1 \
  --gradient_accumulation_steps=4 \
  --max_train_steps=15000 \
  --learning_rate=1e-05 \
  --max_grad_norm=1 \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --output_dir="sd-pokemon-model" \
  --report_to tensorboard

**Comment by flyxiv** [comment]
Project scope is getting big so we split diffusion model into a new repo.
